docfx_yaml/extension.py

```diff
@@ -2086,9 +2106,11 @@ def convert_module_to_package_if_needed(obj):
         file_name_set.add(filename)
 
         for entry in yaml_data:
+            if not app.env.library_shortname:
+                continue
```

Comment:
This would apply to the whole library, right? If so, I think you can just `break` instead of `continue`.

Reply:
You're right, done.